editor/translations/extract.py

```diff
@@ -52,6 +62,36 @@
 """
 
 
+message_patterns = {
+    re.compile(r'RTR\("(([^"\\]|\\.)*)"\)'): False,
+    re.compile(r'TTR\("(([^"\\]|\\.)*)"\)'): False,
+    re.compile(r'TTRC\("(([^"\\]|\\.)*)"\)'): False,
+    re.compile(r'_initial_set\("([^"]+?)",'): True,
+    re.compile(r'GLOBAL_DEF(?:_RST)?\("([^"]+?)",'): True,
+    re.compile(r'EDITOR_DEF(?:_RST)?\("([^"]+?)",'): True,
+    re.compile(r'ADD_PROPERTY\(PropertyInfo\(Variant::[A-Z]+,\s*"([^"]+?)",'): True,
+    re.compile(r'ADD_GROUP\("([^"]+?)",'): False,
+}
+
+
+# See String::camelcase_to_underscore().
+capitalize_re = re.compile(r"(?<=\D)(?=\d)|(?<=\d)(?=\D([a-z]|\d))")
+
+
+def _process_editor_string(name):
+    # See String::capitalize().
+    # fmt: off
+    capitalized = " ".join(
+        part.title()
+        for part in capitalize_re.sub("_", name).replace("_", " ").split()
+    )
+    # fmt: on
```

Comment:
I turned of `black` for this block. Because it wants me to put these on a single line.

Keeping these in multiple lines is much more readable I think.